api/v1alpha1/releaseserviceconfig_types.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const ReleaseServiceConfigResourceName string = "config"
+
+// ReleaseServiceConfigSpec defines the desired state of ReleaseServiceConfig.
+type ReleaseServiceConfigSpec struct {
+	// Debug is the boolean that specifies whether or not the Release Service should run
```

Comment:
The boolean or a boolean?

Reply:
It is "the" boolean that specifies whether or not we run in debug mode. There is no other boolean controlling that. It is the only one

Reply:
It's also "a" flag that enables debug features ;)

But you are the native one here so I trust on your judgement.